developer-docs-site/docs/guides/nfts/mint-nft-cli.md

```diff
@@ -4,587 +4,291 @@ title: "Mint NFTs with the Aptos CLI"
 
 # Mint NFTs with the Aptos CLI
 
-This code lab lets you use the Aptos CLI to mint non-fungible tokens (NFTs) in Aptos so you can see how the process works and employ related functions in your code. Although this code lab assumes use of the devnet network, you could follow the same steps in devnet or even mainnet, assuming you have the necessary funds in your account.
+This tutorial is intended to demonstrate how to programmatically mint NFTs on Aptos. It builds upon the simplest version of an NFT Collection creator manually minting an NFT to a receiver's account to eventually creating a more complex smart contract that facilitates users requesting an NFT and receiving it without ever having to get explicit signer approval from the Collection creator.
 
 ## Prerequisites
 
 This tutorial assumes you have:
 
-* a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
-* the [GitHub CLI](https://cli.github.com/)
 * the [Aptos CLI](../../tools/install-cli/index.md) (or you can run from [aptos-core](https://github.com/aptos-labs/aptos-core) source via `cargo run`)
 * the `aptos-core` repository checked out: `git clone https://github.com/aptos-labs/aptos-core.git`
+* a general understanding of NFTs and NFT Collections
+* an understanding of [the Token V2 standard](https://aptos.dev/guides/nfts/token-v2)
 
-Then:
+## 1. Create a simple smart contract to mint an NFT
 
-* Review the [mint_nft](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/mint_nft) source code and code comments within each subdirectory.
-* Explore the `mint_event_ticket` function defined within each subdirectory.
+We're going to start by making the simplest form of the flow for creating a collection and minting a token and sending it to a user. The code for this part of the tutorial is in the first section of the `move-examples/mint_nft` folder in the aptos-core repository.
 
-## NFT types
+Here are the most basic things we need to do first:
```

Comment:
This sentence sounds a bit redundant. How about "Here are the things we need to do first:"